### ISSUE TITLE
Review packrat dependencies

### ISSUE THREAD

**Issue by mattkerlogue** [ukgovdatascience/govstyle#25]
The `packrat.lock` file seems to have an awfully large number of packages listed for a package that is essentially just providing some `ggplot2` extensions. From my glance through the code very few of them are actually used/needed for the package? And in particular, I'm getting an error when it tries to install the version of `Rcpp` (but I can't seem to work out why that's actually needed based on the package code) - because packrat can't seem to handle compiling from source.

This is what I get out of `packrat::status()

```
The following packages are tracked by packrat, but are no longer available in the local library nor present in your code:
              _       
    assertr     1.0.2 
    base64enc   0.1-3 
    caTools     1.17.1

You can call packrat::snapshot() to remove these packages from the lockfile, or if you intend to use these packages, use packrat::restore() to restore them to your private library.

The following packages are referenced in your code, but are not present
in your library nor in packrat:

    MASS
    Matrix
    bindr
    bindrcpp
    callr
    cli
    clipr
    clisymbols
    desc
    fansi
    fs
    gh
    glue
    ini
    lattice
    mgcv
    nlme
    pillar
    pkgbuild
    pkgconfig
    pkgload
    plogr
    prettyunits
    processx
    ps
    purrr
    rcmdcheck
    remotes
    rlang
    sessioninfo
    tidyselect
    usethis
    utf8
    viridisLite
    xfun
    xopen

You will need to install these packages manually, then use
packrat::snapshot() to record these packages in packrat.

The following packages are used in your code, tracked by packrat, but no longer present in your library:
                    from   to
    Rcpp          0.12.8   NA
    assertthat       0.1   NA
    backports      1.0.4   NA
    bitops         1.0-6   NA
    colorspace     1.3-1   NA
    crayon         1.3.2   NA
    curl             2.3   NA
    devtools      1.12.0   NA
    dichromat      2.0-0   NA
    digest        0.6.10   NA
    dplyr          0.5.0   NA
    evaluate        0.10   NA
    formatR          1.4   NA
    ggplot2        2.2.0   NA
    git2r         0.16.0   NA
    gtable         0.2.0   NA
    highr            0.6   NA
    hms              0.3   NA
    htmltools      0.3.5   NA
    httr           1.2.1   NA
    jsonlite         1.1   NA
    knitr         1.15.1   NA
    labeling         0.3   NA
    lazyeval       0.2.0   NA
    magrittr         1.5   NA
    markdown       0.7.7   NA
    memoise        1.0.0   NA
    mime             0.5   NA
    munsell        0.4.3   NA
    openssl        0.9.5   NA
    packrat      0.4.8-1   NA
    plyr           1.8.4   NA
    png            0.1-7   NA
    praise         1.0.0   NA
    readr          1.0.0   NA
    reshape2       1.4.2   NA
    rmarkdown        1.2   NA
    rprojroot        1.1   NA
    rstudioapi       0.6   NA
    scales         0.4.1   NA
    stringi        1.1.2   NA
    stringr        1.1.0   NA
    testthat       1.0.2   NA
    tibble           1.2   NA
    tidyr          0.6.0   NA
    visualTest     1.0.0   NA
    whisker        0.3-2   NA
    withr          1.0.2   NA
    xtable         1.8-2   NA
    yaml          2.1.14   NA

Use packrat::restore() to restore these libraries.
```

**Comment by mattkerlogue** [comment]
So it seems the offending packages stopping packrat from installing are `Rcpp`, `git2r`, `openssl` and `stringi` which I've manually installed into the local lib. Also getting an error when devtools::check() is run that it needs a more up-to-date version of `knitr` (requires 1.18 or greater, but packrat has it stuck at 1.15.1).

Also roxygen2 not required by packrat, but probably actually should be included to help with documentation.

**Comment by nacnudus** [comment]
I believe this was fixed in #26 by removing packrat altogether. 📦 🐀
